test3/task3/isAccepted.c

```diff
@@ -0,0 +1,73 @@
+#include "isAccepted.h"
+#include <ctype.h>
+
+enum State {
```

Comment:
```suggestion
enum State
{
```
По стайлгайду у нас фигурные скобки одни на строке (если это не инициализатор).